.editorconfig

```diff
@@ -10,5 +10,5 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[{*.json,.eslintrc,.babelrc}]
-indent_size = 2
\ No newline at end of file
+[*.json]
```

Comment:
`.babelrc` doesn't exist here, and existing `.eslintrc` hade a mix of 2 og and 4 space indentation